flink-formats/flink-avro-glue-schema-registry/pom.xml

```diff
@@ -94,6 +94,19 @@ under the License.
 			<groupId>software.amazon.glue</groupId>
 			<artifactId>schema-registry-serde</artifactId>
 			<version>${glue.schema.registry.version}</version>
+			<exclusions>
+				<exclusion>
+					<groupId>com.kjetland</groupId>
+					<artifactId>mbknor-jackson-jsonschema_2.12</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+
+		<!-- The above dependency hard-codes the Scala 2.12 binary version of this jar, so we replace it -->
+		<dependency>
+			<groupId>com.kjetland</groupId>
+			<artifactId>mbknor-jackson-jsonschema_${scala.binary.version}</artifactId>
```

Comment:
@jherico Does this change mean that the format will be compatible with Scala 2.11 also? Although 2.11 has been recently removed